Modules/StudyProgramme/classes/class.ilStudyProgrammeMembersTableGUI.php

```diff
@@ -26,6 +26,7 @@ public function __construct($a_prg_obj_id, $a_prg_ref_id, $a_parent_obj, $a_pare
 
 		$this->prg_obj_id = $a_prg_obj_id;
 		$this->prg_ref_id = $a_prg_ref_id;
+		$this->prg_obj = new ilObjStudyProgramme($a_prg_ref_id);
```

Comment:
I'm quite sure that you don't need to initialize the object here. You should be able to get if from elsewhere (like parent_object).